gradle/dependencies.gradle

```diff
@@ -15,10 +15,11 @@
  */
 
 def versions = [
-        kotlin        : '1.4.20',
+        kotlin        : '1.6.21',
```

Comment:
This is still not super modern - but attempting to updating it to 1.8 generated some `kotlin-android-extensions` and I didn't want to commit to 1.8 in this pull request in case it was not trivial to fix those errors in this repository.